src/api/profile/profile.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+import { ENDPOINT } from "../../utils/urls";
+import { getAuthToken } from "../../api/auth/login";
+
+export async function profile({ token }) {
```

Comment:
Is there a reason to descontruct the object token here? Why we are not using just (token)?